ts/features/bonus/bpd/components/superCashbackRanking/SuperCashbackHeader.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { Badge, Text, View } from "native-base";
+import { StyleSheet } from "react-native";
+import { fromNullable } from "fp-ts/lib/Option";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { bpdSelectedPeriodSelector } from "../../store/reducers/details/selectedPeriod";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import { IOColors } from "../../../../../components/core/variables/IOColors";
+import { BpdPeriodWithInfo } from "../../store/reducers/details/periods";
+import I18n from "../../../../../i18n";
+import { H3 } from "../../../../../components/core/typography/H3";
+
+type Props = ReturnType<typeof mapStateToProps>;
+
+const styles = StyleSheet.create({
+  badgeAqua: {
+    backgroundColor: IOColors.aqua,
+    height: 18,
+    marginTop: 4
+  },
+  badgeText: {
```

Comment:
Did you check if on Android it has a good alignment?
https://github.com/pagopa/io-app/blob/0cff18243111ae496a808a55091c7ccbe83b7311/ts/components/wallet/PaymentMethodsList.tsx#L73